src/oracle_instance_manager.ts

```diff
@@ -137,6 +143,20 @@ export default class OracleInstanceManager {
                 metadata: { group: groupName },
             };
             await this.jibriTracker.track(ctx, state);
+            if (isScaleDownProtected) {
+                await this.shutdownManager.setScaleDownProtected(
+                    ctx,
+                    {
+                        instanceId: launchResponse.instance.id,
+                        instanceType: 'jibri',
```

Comment:
Is this safe to hardcode as 'jibri' ?  Shouldn't this be set from to the type property on the group?